src/vaadin-combo-box-light.html

```diff
@@ -150,6 +163,20 @@
           this.inputElement[this._propertyForValue] = value;
         }
       }
+
+      /**
+       * Focusable element used by vaadin-control-state-mixin
+       */
+      /*
```

Comment:
Kind reminder to remove the commented code.

Reply:
Oops. Thanks for noticing. Will do. 😄

Reply:
Done.